fedora/client/bodhi.py

```diff
@@ -243,11 +244,25 @@ def query(self, **kwargs):
         if 'mine' in kwargs:
             kwargs['user'] = self.username
         if 'package' in kwargs:
-            kwargs['packages'] = kwargs['package']
+            # for Bodhi 1, 'package' could be a package name, build, or
+            # update ID, so try and figure it out
+            if re.search(r'(\.el|\.fc)\d\d?', kwargs['package']):
+                kwargs['builds'] = kwargs['package']
+            elif re.search(r'FEDORA-(EPEL)?-\d{4,4}', kwargs['package']):
+                kwargs['updateid'] = kwargs['package']
+            else:
+                kwargs['packages'] = kwargs['package']
             del(kwargs['package'])
         if 'release' in kwargs:
             kwargs['releases'] = kwargs['release']
             del(kwargs['release'])
+        if 'type_' in kwargs:
+            kwargs['type'] = kwargs['type_']
+            del(kwargs['type_'])
+        # Old Bodhi CLI set bugs default to "", but new Bodhi API
+        # checks for 'if bugs is not None', not 'if not bugs'
+        if kwargs['bugs'] == "":
```

Comment:
sure, you're right, I just forgot to do that. It should follow all the others.

As for the other thing, my thinking was exactly as pypingou said, but it wouldn't hurt to do it the other way I guess.